cypress/integration/pages/summary.spec.js

```diff
@@ -49,4 +56,15 @@ context('Summary page', () => {
       });
     });
   });
+
+  describe('Action list', () => {
+    it('Can render a clickable link in action description', () => {
+      cy.get('[data-testid=details-summary]').click({ force: true });
+
+      cy.get('[data-testid=action-details-test] > details > div > p >  a')
```

Comment:
Could we add a test id to the final `a` in this selector instead of chaining things maybe? This makes the tests a bit brittle when it comes to updating the page layout.

Reply:
The a tag gets generated within <ReactMarkdown /> so we can't really access it from within the code to add test id

Reply:
Ah yeah of course, forgot what we were testing! This is cool then.